src/remoteRunner.js

```diff
@@ -1,53 +1,16 @@
-import crypto from 'crypto';
-import fs from 'fs';
-import os from 'os';
-import path from 'path';
-
-import Archiver from 'archiver';
-
-import { FILE_CREATION_DATE } from './createStaticPackage';
 import Logger, { logTag } from './Logger';
 import constructReport from './constructReport';
 import createHash from './createHash';
 import ensureTarget from './ensureTarget';
 import loadCSSFile from './loadCSSFile';
 import uploadAssets from './uploadAssets';
-import validateArchive from './validateArchive';
-
-function staticDirToZipFile(dir) {
-  return new Promise((resolve, reject) => {
-    const archive = new Archiver('zip', {
-      // Concurrency in the stat queue leads to non-deterministic output.
-      // https://github.com/archiverjs/node-archiver/issues/383#issuecomment-2253139948
-      statConcurrency: 1,
-      zlib: { level: 6 },
-    });
-
-    const rnd = crypto.randomBytes(4).toString('hex');
-    const pathToZipFile = path.join(os.tmpdir(), `happo-static-${rnd}.zip`);
-    const output = fs.createWriteStream(pathToZipFile);
-    const entries = [];
-
-    archive.on('entry', (entry) => {
-      entries.push(entry);
-    });
-
-    output.on('close', async () => {
-      validateArchive(archive.pointer(), entries);
-      resolve(pathToZipFile);
-    });
-    archive.pipe(output);
-    archive.directory(dir, false, { date: FILE_CREATION_DATE });
```

Comment:
I believe archive.directory may be nondeterministic, so this should be a good improvement for folks using remote runner.